privacyidea/models/audit.py

```diff
@@ -24,7 +24,7 @@
 from privacyidea.models.utils import MethodsMixin
 from privacyidea.lib.utils import convert_column_to_unicode
 
-audit_column_length = {"signature": 620,
+audit_column_length = {"signature": 1132,
```

Comment:
I think the extra 93 characters were some kind of upper bounds in case we switched signing methods. So rounding this to 1100 should be ok.